bentoml/utils/usage_stats.py

```diff
@@ -86,6 +87,10 @@ def track_save(bento_service):
         return track("save", info)
 
 
+def track_load_start():
+    return track('load-start', {})
```

Comment:
let's move the logic of checking `if config['core'].getboolean("usage_tracking"):`  to track function above :)

Reply:
otherwise it's easy to forget

Reply:
Yep. Good catch